examples/nuxt-basic/README.md

```diff
@@ -34,6 +34,21 @@ You can read more about creating the service account JSON file and setting up th
 
 > Important! Node v10 is currently NOT supported
 
+### (optional) Configure Firestore DB
+
+_You could skip this step if you your Realtime Database._
```

Comment:
small typo here. `your` should probably be `use`

Reply:
nice catch, fixed it :)